### DISCUSSION TITLE
Opacity property with Camera object seems broken.

### DISCUSSION THREAD

**Discussion by Kilanvl1** [motion-canvas/motion-canvas#1141] [Q&A]
Head over to: https://motioncanvas.io/api/2d/components/Camera and in the code editor showcasing the Camera node, add the following line: yield* rect().opacity(0,2) right after the view is created. 

You should see the rectangle immediately disappear instead of animating the opacity to 0 over 2 seconds. If you track the opacity value over the 2 seconds, you see that it is gradually decreasing but seems to disappear the moment it's less than 1. 

Is this expected behaviour, or is it just a bug?

**Comment by rileywrites** [comment]
i have the same issue!

EDIT: using @aarthificial's workaround in https://github.com/motion-canvas/motion-canvas/issues/1057 fixes the problem!
> It seems to be a problem with the cache calculations. Until the fix is ready, you can set the position of the camera's scene to half the size of the view:
> 
> ```ts
> export default makeScene2D(function* (view) {
> 	const camera = createRef<Camera>();
> 	const browserRef = createRef<Img>();
> 
> 	view.add(
> 		<Camera ref={camera}>
> 			<Img ref={browserRef} src={browser} height={400} width={1000} position={[0, 0]} shadowBlur={10} shadowColor="#0004" />
> 		</Camera>,
> 	);
> 	
> 	// Do this after creating the camera:
> 	camera().scene().position(view.size().div(2));
> 
> 	yield* waitFor(10)
> });
> ```
> 
> This should fix the issue for now.
